standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1895,6 +1895,10 @@ public enum ConfVars {
             "hive.metastore.httpserver.threadpool.idle", 60_000L,
             "HMS embedded HTTP server thread idle time."
     ),
+    HTTPSERVER_USE_HTTPS("hive.metastore.httpserver.use.https",
```

Comment:
why do we need new config? can't we reuse `USE_SSL`?

Reply:
I did not want to presume that USE_SSL would imply https for Iceberg Catalog and Properties; routing to these endpoints using https may imply more configuration on the client side (truststore).
We could use USE_SSL as default if ...use.https is not explicitly set ?